tuneme/templates/new/base.html

```diff
@@ -59,12 +59,12 @@ <h2 class="header__search-title">{% trans "Do you have questions on love, sex an
     <ul class="menu-items">
       <li class="menu__item menu__item--default"><a href="#top">{% trans "Back to top" %}</a></li>
       <li class="menu__item menu__item--default"><a href="/">{% trans "Home" %}</a></li>
-      {% if 'tuneme.org' in request.get_host %}
+      <!-- {% if 'tuneme.org' in request.get_host %}
```

Comment:
use `{% comment %} ... {% endcomment %}` instead.
`<!-- ... -->` is only really good for HTML only